backend/src/process/TagProcess.py

```diff
@@ -0,0 +1,45 @@
+from uuid import UUID
+
+from utils.HttpResponse import HttpResponse, get_with_data, get_with_error
+from command.TagCommands import remove_tag
+from query.TagQueries import get_tags, get_tag_by_id
+from validation.Validation import validate_short_id
+
+
+def handle_get_tags() -> HttpResponse:
+    tags = get_tags()
+    tags_json = {}
+    for tag in tags:
+        tags_json[str(tag.tag_id)] = tag.to_json()
+
+    return get_with_data({
+        'tags': tags_json
+    })
+
+
+def handle_get_tag_by_id(tag_id: str) -> HttpResponse:
+
+    short_id_res = validate_short_id(tag_id)
+    if short_id_res.is_error:
+        return get_with_error(400, short_id_res.message)
```

Comment:
In general, perhaps defined constant variables for statuscodes (e.g. `STATUS_BAD_REQUEST = 400`) to avoid using numbers directly in the code.